runtime/runtime/src/config.rs

```diff
@@ -267,7 +270,16 @@ pub fn tx_cost(
     // If signer is equals to receiver the receipt will be processed at the same block as this
     // transaction. Otherwise it will processed in the next block and the gas might be inflated.
     let initial_receipt_hop = if transaction.signer_id == transaction.receiver_id { 0 } else { 1 };
-    let minimum_new_receipt_gas = fees.min_receipt_with_function_call_gas();
+    let minimum_new_receipt_gas =
+        if !checked_feature!("stable", DecreaseFunctionCallBaseCost, protocol_version) {
+            fees.min_receipt_with_function_call_gas()
+        } else {
+            // The pessimistic gas pricing is a best-effort limit which can be breached in case of
+            // congestion when receipts are delayed before they execute. Hence there is not much
+            // value to tie this limit to the function call base cost. Making it constant limits
+            // overcharging to 6x, which was the value before the cost increase.
+            4_850_000_000_000 // 4.85TGas.
```

Comment:
In general, I agree with introducing dedicated constants. In this particular case, I don't think there is much value to be gained by doing this - we don't plan to reuse it anywhere (if we need to, adding a constant would make sense, otherwise, YAGNI applies), we already have a docstring that explains it and seeing it in a separate file without a usage context will yield very little insight to the reader and will introduce unnecessary indirection when reading this code.